app/views/users/_user_info.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="user-info-container">
+    <div class="bio">
+        <h2>Bio</h2>
+        <p><%= user.bio %></p>
+    </div>
+    <% counter = 0 %>
+    <% user.posts.each_with_index do |post, idx| %>
+        <% if counter < 2 %>
+            <div class="post">
+            <h2>Post: <%= post.title %></h2>
+            <p><%= post.text %></p>
+            </div>
+            <% counter += 1 %>
+        <% else %>
+            <% break %>
+        <% end %>
+    <% end %>
+    <button id="see-all">See all posts</button>
```

Comment:
- This button should link to the post#index page. Please 🙏🏽 add the function to the button to be able to see all posts.